gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -1405,14 +1406,20 @@ private static void checkContentType(BlobInfo blobInfo) throws IllegalArgumentEx
   BatchResponse apply(BatchRequest batchRequest);
 
   /**
-   * Return a channel for reading the blob's content.
+   * Return a channel for reading the blob's content. The blob's latest generation is read. If the
```

Comment:
No, I described it incorrectly.

Basically I was trying to suggest that if one uses `BlobSourceOptions.generationMatch(long)` or `BlobSourceOptions.generationMatch()` when `BlobId` has a generation value then in effect that
also guaranty to either read from that version or fail. Though if one does not specify the generation as part of the `BlobId` the generation that will be used is latest.

I am not sure how to phrase that level of detail without confusing. If you can think of something, great. If not, lets merge it as is.